### ISSUE TITLE
194 landmark shape predictor 

### ISSUE THREAD

**Issue by dominic1611** [davisking/dlib#2064]
hi, im currently working on training a shape predictor by using a dataset of 194 landmarks.As my project only requires the mouth landmarks. May I know which landmarks of the mouth is positioned at?

**Comment by arrufat** [comment]
If you use dlib's pretrained shape predictor for face landmarks, you should check this to know what each number refers to:
https://github.com/davisking/dlib/blob/master/dlib/image_processing/render_face_detections.h#L36-L77

**Comment by dominic1611** [comment]
hi im using the shape predictor with 194 landmarks instead of this standard 68 one . hence the same facial features will have different landmark number in both 194 and 68 shape predictors.

**Comment by xjohnxjohn** [comment]
@dominic1611 What the dataset of 194 landmarks you used? Do you had the details of 194 landmarks? Have any links?

**Comment by dominic1611** [comment]
Hi i got my training dataset here
https://stackoverflow.com/questions/36711905/dlib-train-shape-predictor-ex-cpp

Its somewhere in the comments

**Comment by dexception** [comment]
Please don't use LBP. The accuracy is pathetic and should never be used for production.
